src/main/java/com/codeup/adlister/dao/ListAdsDao.java

```diff
@@ -8,61 +8,61 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class ListAdsDao {
-    private List<Ad> ads;
-
-    public List<Ad> all() {
-        if (ads == null) {
-            ads = generateAds();
-        }
-        return ads;
-    }
-
-    public Long insert(Ad ad) {
-        // make sure we have ads
-        if (ads == null) {
-            ads = generateAds();
-        }
-        // we'll assign an "id" here based on the size of the ads list
-        // really the dao would handle this
-        ad.setId((long) ads.size());
-        ads.add(ad);
-        return ad.getId();
-    }
-
-    private List<Ad> generateAds() {
-        List<Ad> ads = new ArrayList<>();
-        ads.add(new Ad(
-            1,
-            1,
-            "playstation for sale",
-            "This is a slightly used playstation",
-            35
-
-        ));
-        ads.add(new Ad(
-            2,
-            1,
-            "Super Nintendo",
-            "Get your game on with this old-school classic!",
-            10
-
-        ));
-        ads.add(new Ad(
-            3,
-            2,
-            "Junior Java Developer Position",
-            "Minimum 7 years of experience required. You will be working in the scripting language for Java, JavaScript",
-            40000
-        ));
-        ads.add(new Ad(
-            4,
-            2,
-            "JavaScript Developer needed",
-            "Must have strong Java skills",
-            100000
-        ));
-        return ads;
-    }
-
-}
+//public class ListAdsDao {
```

Comment:
In cases like this it is better to remove the file rather than commenting it out. If you need it back at some point you can always get it from a commit using git.